docs-source/content/samples/simple/elastic-stack/_index.md

```diff
@@ -4,33 +4,3 @@ date: 2019-02-23T17:32:31-05:00
 weight: 6
 description: "Sample for configuring the Elasticsearch and Kibana deployments and services for the operator's logs."
```

Comment:
update this text - this shows up in the toc page, up a level.  make it say "samples" ... "WebLogic domains and operator's logs"

Reply:
done in next push